src/taggle.js

```diff
@@ -631,4 +631,18 @@
 
     window.Taggle = Taggle;
 
+    /* global define, module */
+    if ( typeof define === 'function' && define.amd ) {
+        // AMD
+        define([], function () {
+            return Taggle;
+        });
+    } else if (typeof exports === 'object') {
+        // CommonJS
+        module.exports = Taggle;
+    } else {
+        // Vanilla browser global
+        window.Taggle = Taggle;
```

Comment:
hey @Schoonology sorry this took so long. Can you rebase on top of master and fix the conflicts? Also it's probably not useful to have `window.Taggle = Taggle` twice right?